src/main/java/org/cloudburstmc/server/player/OfflinePlayer.java

```diff
@@ -147,12 +141,12 @@ public Player getPlayer() {
     }
 
     @Override
-    public Long getFirstPlayed() {
+    public long getFirstPlayed() {
         return this.namedTag != null ? this.namedTag.getLong("firstPlayed") : null;
     }
 
     @Override
-    public Long getLastPlayed() {
+    public long getLastPlayed() {
         return this.namedTag != null ? this.namedTag.getLong("lastPlayed") : null;
```

Comment:
This function is still returning a null value so it will fail to build.

Reply:
Oh, i didn't even spot that. Makes sense why it wasnt a primitive